src/Compilers/CSharp/Portable/Binder/ImportChain.cs

```diff
@@ -13,13 +13,16 @@
 namespace Microsoft.CodeAnalysis.CSharp
 {
     [DebuggerDisplay("{GetDebuggerDisplay(),nq}")]
-    internal sealed class ImportChain : Cci.IImportScope
+    internal sealed class ImportChain : Cci.IImportScope, IImportChain
     {
         public readonly Imports Imports;
         public readonly ImportChain ParentOpt;
 
         private ImmutableArray<Cci.UsedNamespaceOrType> _lazyTranslatedImports;
 
+        private ImmutableArray<IAliasSymbol> _lazyImportChainAliases;
+        private ImmutableArray<INamespaceOrTypeSymbol> _lazyImportChainImports;
```

Comment:
We probably do not want to cache these arrays, at least not within this type.

Reply:
SGTM.  I no longer cache these.  IDE doesn't see a need for it either.